bidwire/alembic/versions/9b30b0fe231a_create_documents_table.py

```diff
@@ -23,6 +23,7 @@ def upgrade():
         sa.Column('url', sa.String, unique=True),
         sa.Column('title', sa.String),
         sa.Column('site', sa.String),
+        sa.Column('description', sa.String),
```

Comment:
PS When you create the new revision, make sure that you are fully synced with master, so that the `down_revision` field in your newly-created migration points to the most recent revision at master.